src/main/kotlin/com/autonomousapps/internal/utils/coordinatesStrings.kt

```diff
@@ -34,8 +34,16 @@ internal fun <T> String.matchesKey(mapEntry: Map.Entry<String, T>): Boolean {
   return false
 }
 
-internal fun <T> String.equalsKey(mapEntry: Map.Entry<String, T>) =
-  mapEntry.key.firstCoordinatesKeySegment() == this || mapEntry.key.secondCoordinatesKeySegment() == this
+internal fun <T> String.equalsKey(mapEntry: Map.Entry<String, T>): Boolean {
```

Comment:
please note, that this method is also used is `ReasonTask.getUsagesFor`. Can it affect the logic here?